### DISCUSSION TITLE
prime react diamond theme styles not working properly from version ^10.0.0

### DISCUSSION THREAD

**Discussion by Sakshi-Antala** [primefaces/community#1816] [PrimeReact Templates] [answered]
I integrate diamond theme in my react project. Its working fine till primereact version 9.6.2 but after 10.0.0 version it showing double checkbox ,radio button.css not working properly and checkbox is not highlighted with on click
![image](https://github.com/primefaces/community/assets/118065394/ce86f21f-91d0-44fe-a115-4af496e578d4)

**Comment by melloware** [comment, marked as answer]
See these instructions for how to Update DIAMOND so your CSS is correct for PrimeReact 10.6.4: https://github.com/primefaces/primereact/wiki/Migration-Guide-for-Templates

**Reply by boyabenyathil** [reply]
im facing same problem with apolo 10.1.0, i have ran command sass --update public/theme:public/theme still no changes
